.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,23 @@
+name: rubocop
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "0 21 * * 6"
```

Comment:
This one I don't think benefits from a scheduled run does it? Is there a good reason to run rubocop on a schedule?

Reply:
Yes. Check that everything for rubocop is installing and running. Once per week is good trade off.

Reply:
Even better. We will know that this GiHub Action still working.

Reply:
Ok. We'll see how it goes